wikiconv/README.md

```diff
@@ -1,8 +1,8 @@
 # WikiConv
 
-WikiConv is corpus encompassing the full history of conversations on Wikipedia.
+WikiConv is a corpus encompassing conversations happens on Wikipedia.
 
-The dataset and reconstruction process for this corpus has been published in the paper [WikiConv: A Corpus of the Complete Conversational History of a Large Online
+The dataset and reconstruction process for the corpra has been published in the paper [WikiConv: A Corpus of the Complete Conversational History of a Large Online
```

Comment:
I think this should be `the corpus` (corpra is a typo, and corpora is for the plural)